README.md

```diff
@@ -3,3 +3,49 @@
 [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/ProjectPythia/pythia-tutorials/HEAD)
 
 A single repository to host community-contributed Pythia tutorial content (notebooks, data, and markdown).
+
+The goal of this repository is to create a common location and environment for Pythia Python tutorials. For the Python learner, this has the benefits of consistent installation / spin up before jumping in to each lesson. For the Pythia team, this ownership allows us to maintain content long-term so that we can fix things when they break.
+
+We, the Pythia team, want there to be a low barrier to entry for the tutorial speaker to develop and host their tutorial content in this repository. For the speaker's convenience we will provide a simple template. This repository differs from `pythia-foundations` book, which has a high barrier for entry, with its strict and consistent style guide and rigorous review process before merging. Further, the tutorial notebooks in this repository are designed to be accompanied by a live speaker or recording. This means that the amount of explanatory text between code cells or excercises may differ between tutorials based on the tutorial speaker and developer's preferred style.
```

Comment:
perhaps change "want there to be a low barrier to entry for the tutorial speaker" to "we want to lower the barrier of entry for the tutorial speaker"?